docker-compose.yml

```diff
@@ -108,3 +108,13 @@ services:
       - ./logs/api_nginx:/var/log/nginx
     ports:
       - "8055:80"
+          
+  website:
+    build: './website/client'
+    container_name: wow-collection-website
+    volumes:
+    - './website/client:/home/node/app'
```

Comment:
Instead of /home/node/app use something similar to the other services (ex : /api for the api). You could use /website or a classic as /app

Reply:
Same tonight